docs/data/material/getting-started/example-projects/ExampleCollection.js

```diff
@@ -118,17 +119,20 @@ export default function ExampleCollection() {
               src={example.src}
               alt={example.name}
             />
-            <span>
+            <div>
               <Typography
                 variant="body"
                 fontWeight={600}
                 sx={{ fontFamily: 'IBM Plex Sans' }}
               >
                 {example.name}
               </Typography>
-              <Stack
-                direction={{ xs: 'column', sm: 'row' }}
-                alignItems={{ xs: 'start', sm: 'center' }}
+              <Box
+                sx={{
+                  display: 'flex',
+                  flexWrap: 'wrap',
+                  alignItems: 'baseline',
+                }}
```

Comment:
@siriwatknp Hey! Just for the sake of learning, I'm curious why the preference for using the `Box` component rather than the `Stack`?